### DISCUSSION TITLE
disable env var detection

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei-templates#3067] [Q&A] [answered]
Hi team!

Question: what if my payload contains `{{payload here}}` and that is not cli env var. Can i escape it or somehow disable nuclei vars detection ?

Thanks

**Comment by ehsandeep** [comment, marked as answer]
@savushkin-yauheni `skip-variables-check: true`

Example template:- https://github.com/projectdiscovery/nuclei-templates/blob/a21cec63625b1bc4392d67e432f54f1453f91c94/vulnerabilities/other/pdf-signer-ssti-to-rce.yaml#L16
